### ISSUE TITLE
Upgrade to Angular 10

### ISSUE THREAD

**Issue by Quraian** [ng-bootstrap/ng-bootstrap#3787]
[Angular 10 is already out](https://blog.angular.io/version-10-of-angular-now-available-78960babd41)

**Comment by thetric** [comment]
Might be fixed in https://github.com/ng-bootstrap/ng-bootstrap/pull/3790

**Comment by santam85** [comment]
@thetric I believe github had some stability issues this morning, didn't update that PR when I added new commits  🤔

**Comment by maxokorokov** [comment]
We'll be releasing the ng-bootstrap 7.0.0 with Angular 10 support in the comings days as soon as we fix the schematics issues and double-check everything.

**Comment by santam85** [comment]
@maxokorokov Schematics issues seem to be fixed for me by #3802

**Comment by tillias** [comment]
Any updates on this issue? I'm only angular 10, but have to use bootstrap 6.1.0

```
npm WARN @ng-bootstrap/ng-bootstrap@6.1.0 requires a peer of @angular/common@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-bootstrap/ng-bootstrap@6.1.0 requires a peer of @angular/core@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-bootstrap/ng-bootstrap@6.1.0 requires a peer of @angular/forms@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-bootstrap/ng-bootstrap@6.1.0 requires a peer of @angular/localize@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-bootstrap/ng-bootstrap@6.1.0 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
```
